.github/workflows/publish.yml

```
@@ -32,23 +32,23 @@ jobs:
       - name: Login to Docker Hub
         uses: docker/login-action@v1
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          username: ${{ secrets.DOCKERHUB_READ_USER }}
+          password: ${{ secrets.DOCKERHUB_READ_TOKEN }}
 
-      - name: Publish docker image
+      - name: publish docker image
         run: ./gradlew publish dockerPushImages
         env:
-          DOCKER_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_TOKEN }}
+          username: ${{ secrets.DOCKERHUB_PUBLISH_USER }}
+          password: ${{ secrets.DOCKERHUB_PUBLISH_TOKEN }}
```

This step also needs the bintray creds, added below:
```suggestion
          DOCKER_USERNAME: ${{ secrets.DOCKERHUB_PUBLISH_USER }}
          DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_PUBLISH_TOKEN }}
          ORG_GRADLE_PROJECT_publishUser: ${{ secrets.BINTRAY_USER }}
          ORG_GRADLE_PROJECT_publishApiKey: ${{ secrets.BINTRAY_API_KEY }}
```